Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -149,23 +149,111 @@ Developers should take care to secure sensitive IPC components with the `signatu
 
 #### Overview
 
--- TODO [Provide a general description of the issue.] --
+Android apps can expose functionality to:
+* other apps via IPC mechanisms,
```

Comment:
Maybe it is better to mention them here:
- Intents
- Binders
- ASHMEM
- BroadcastReceivers